README.md

```diff
@@ -27,17 +27,31 @@ runtime: python3 # Defines the runtime environment for the Structure
 runtime_version: 3.11 # Defines the specific version of the runtime environment for the Structure
 build: # Defines the build-time configuration for the Structure
   requirements_file: requirements.txt # Defines the path to the requirements.txt file for the Structure, relative to the structure_config.yaml. Or absolute from the repository root if a forward slash is used: `/requirements.txt`.
+  cache_build_dependencies: # Defines the configuration for caching build dependencies in order to speed up Deployments
+    enabled: false # Defines whether the build dependency caching is on or off
+    watched_files: # Defines the particular files that will trigger cache invalidation, resulting in a full rebuild of the Structure and dependencies
+      - requirements.txt
 run: # Defines the run-time configuration for the Structure
   main_file: structure.py # Specifies the path to the entry point file of the Managed Structure. This path is relative to the structure_config.yaml. Or absolute from the repository root if a forward slash is used: `/structure.py`.
 ```
 
+#### Cache Build Dependencies Field
+
+By default, Griptape Cloud will rebuild and reinstall all dependencies for a Structure on Deployment.
+
+When `cache_build_dependencies` is disabled, Griptape Cloud will always build the Structure code and install dependencies on every Deployment.
+
+When `cache_build_dependencies` is enabled, Griptape Cloud will cache the base runtime dependencies for the Structure for reuse on subsequent Deployments. That cache will be invalidated when any of the configured `watched_files` change. For instance, if the `requirements.txt` file is configured as the only `watched_file`, then Griptape Cloud will only rebuild the Structure and install dependencies when that file has changes from a previous Deployment. If there are no changes to `requirements.txt`, then Griptape Cloud will reuse the previously built base Structure runtime, and copy over the new Deployment's Structure code files. This speeds up the Deployment process, especially for Structures with many dependencies.
```

Comment:
Can we move the sentence `This speeds up the Deployment process, especially for Structures with many dependencies.` to be right after `When `cache_build_dependencies` is enabled, Griptape Cloud will cache the base runtime dependencies for the Structure for reuse on subsequent Deployments.` so we can get the value up front sooner?